docs/CHANGELOG/CHANGELOG-1.5.md

```diff
@@ -0,0 +1,117 @@
+- karmadactl: Introduced `--cert-validity-period` for `init` to make the validity period of cert configurable. ([#3156](https://github.com/karmada-io/karmada/pull/3156), @lonelyCZ)
+- `karmada-controller-manager`: Now the `OverridePolicy` and `ClusterOverridePolicy` will be applied by implicit priority order. The one with the lower priority will be applied before the one with the higher priority. ([#2609](https://github.com/karmada-io/karmada/pull/2609))
+- `karmada-controller-manager`: Users are now able to apply multiple dependencies interpreter configurations. ([#2884](https://github.com/karmada-io/karmada/pull/2884), @XiShanYongYe-Chang)
+- `karmada-controller-manager`: Built-in interpreter supports StatefulSets. ([#3009](https://github.com/karmada-io/karmada/pull/3009), @chaunceyjiang)
+- `karmada-controller-manager`:  Retained the labels added to resources by member clusters. ([#3088](https://github.com/karmada-io/karmada/pull/3088), @chaunceyjiang)
+- `karmada-controller-manager`: Default interpreter supports CronJob aggregated status. ([#3129](https://github.com/karmada-io/karmada/pull/3129), @chaunceyjiang)
+- `karmada-controller-manager`: Support for removing annotations/labels propagated through karmada. ([#3099](https://github.com/karmada-io/karmada/pull/3099), @chaunceyjiang)
+- `karmada-webhook`: Added validation for policy.spec.placement.orderedClusterAffinities. (#3164, @XiShanYongYe-Chang)
+- `karmada-webhook`: Validated the fieldSelector of overridepolicy. ([#3193](https://github.com/karmada-io/karmada/pull/3193), @chaunceyjiang)
+
+## Other
+### Dependencies
+- Kubernetes images will now be pulled from registry.k8s.io instead of k8s.gcr.io, to be in alignment with current community initiatives. ([#2882](https://github.com/karmada-io/karmada/pull/2882), @Zhuzhenghao)
+- Karmada is now built with Golang 1.19.4. ([#2908](https://github.com/karmada-io/karmada/pull/2908), @qingwave)
+- Karmada is now built with Golang 1.19.5. ([#3067](https://github.com/karmada-io/karmada/pull/3067), @yanggangtony)
+- Karmada is now built with Kubernetes v1.26.1 dependencies. ([#3080](https://github.com/karmada-io/karmada/pull/3080), @RainbowMango)
+- The base image `alpine` now has been promoted from `alpine:3.15.1` to `alpine:3.17.1`. ([#3045](https://github.com/karmada-io/karmada/pull/3045), @fengshunli)
+
+### Helm Chart
+- Fixed helm template missing yaml directive marker to separate API Service resources. ([#2963](https://github.com/karmada-io/karmada/pull/2963), @a7i)
+- Fixed missing karmada-search helm template strategy. ([#2994](https://github.com/karmada-io/karmada/pull/2994), @a7i)
+- Fixed karmada-agent helm template strategy indentation. ([#2993](https://github.com/karmada-io/karmada/pull/2993), @a7i)
+- Supports PodDisruptionBudget resource in default interpreter. ([#2997](https://github.com/karmada-io/karmada/pull/2997), @a7i)
```

Comment:
> Supports PodDisruptionBudget resource in default interpreter. 

Shouldn't this be in `Features & Enhancements`?

Reply:
right.